### ISSUE TITLE
ImportError: No JS module 'tweener'

### ISSUE THREAD

**Issue by lalten** [pbxqdown/gnome-shell-extension-transparent-window#11]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Arch
- **GNOME version**: 3.38.1
- **Exact command to reproduce**: Activate "On" slider on https://extensions.gnome.org/extension/1454/transparent-window/

### Describe the problem
Describe the problem here, including the steps to reproduce the problem.

Installing fails, "On" slider displays "ERROR".

### Source code / logs 
Include any logs or source code that would be helpful to diagnose the problem.   
First, in the preference, Set the log verbose level to "Debug";  

`journalctl -f` has this entry:
```
Nov 07 15:35:18 tpxy gnome-shell[922]: JS ERROR: Extension transparent-window@pbxqdown.github.com: ImportError: No JS module 'tweener' found in search path
                                       @/home/myuser/.local/share/gnome-shell/extensions/transparent-window@pbxqdown.github.com/extension.js:3:17
                                       _callExtensionInit@resource:///org/gnome/shell/ui/extensionSystem.js:420:13
                                       _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:145:18
                                       _onEnabledExtensionsChanged/<@resource:///org/gnome/shell/ui/extensionSystem.js:491:35
                                       _onEnabledExtensionsChanged@resource:///org/gnome/shell/ui/extensionSystem.js:491:14
                                       createCheckedMethod/<@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:533:46
                                       enableExtension@resource:///org/gnome/shell/ui/extensionSystem.js:194:29
                                       callback@resource:///org/gnome/shell/ui/extensionDownloader.js:223:44
                                       gotExtensionZipFile/<@resource:///org/gnome/shell/ui/extensionDownloader.js:106:13
```

**Comment by pbxqdown** [comment]
Thanks for submitting the issue. It has been fixed by MR #10. New version would be available soon after reviewed by GNOME extension store.

**Comment by lalten** [comment]
Oh, I missed that. Thanks!
